tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -1034,6 +1034,8 @@ class StructureAnnotator : public ide::SyntaxModelWalker {
     for (auto &Elem : Node.Elements) {
       tagRange(Elem.Range, getTagName(Elem.Kind), Node);
     }
+    if (Node.TypeRange.isValid() && Node.Range.contains(Node.TypeRange))
```

Comment:
this is needed because an explicit `value` parameter in a `subscript` `set` has a type range which is the subscript return type (i.e. it's outside the node itself).

Example:

```swift
subscript(string: String) -> Int {
  get {
	return 0
  }
  set(value) {
	print(value)
  }
}
```